### PR TITLE
MF-219: Clean up translation strings

### PR DESCRIPTION
**Pull request by denniskigen**
https://issues.openmrs.org/browse/MF-219

This PR makes a bunch of refactors to chart summary widgets so that all the translation keys are in **camelCase** format.

### CODE REVIEW

File: src/widgets/appointments/appointments-detailed-summary.component.tsx [hunk 1]
```diff
@@ -109,12 +112,15 @@ export default function AppointmentsDetailedSummary(
...
           showComponent={() =>
-            openWorkspaceTab(AppointmentsForm, `${t("Appointments Form")}`)
+            openWorkspaceTab(
+              AppointmentsForm,
+              `${t("appointmentsForm", "Appointments Form")}`
+            )
           }
           addComponent={AppointmentsForm}
-          displayText="This patient has no appointments recorded in the system."
+          displayText={t("appointments_lower", "appointments")}
```
Comment (brandones) [inline src/widgets/appointments/appointments-detailed-summary.component.tsx]: upper/lower should be handled with CSS [text-transform](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform)

File: src/widgets/appointments/appointments-form.test.tsx [hunk 2]
```diff
@@ -80,7 +80,7 @@ describe("<AppointmentsForm />", () => {
       </BrowserRouter>
     );
 
-    await screen.findByText("Schedule New Appointment");
+    await screen.findByText("Schedule new appointment");
```
Comment (brandones) [inline src/widgets/appointments/appointments-form.test.tsx]:
```suggestion
    await screen.findByText(/schedule new appointment/i);
```

File: src/widgets/immunizations/immunizations-detailed-summary.component.tsx [hunk 3]
```diff
@@ -159,13 +159,13 @@ export default function ImmunizationsDetailedSummary(
...
-            <Trans i18nKey="no immunization configured">
-              No Immunizations are configured.
+            <Trans i18nKey="noImmunizationsAreConfigured">
+              No immunizations are configured.
             </Trans>
           </p>
           <p className="omrs-medium">
             <a href="https://github.com/openmrs/openmrs-esm-patient-chart-widgets#configuration">
-              <Trans i18nKey="please configure immunization">
+              <Trans i18nKey="pleaseConfigureImmunizations">
```
Comment (brandones) [inline src/widgets/immunizations/immunizations-detailed-summary.component.tsx]: This copy seems likely to change. How about

```suggestion
              <Trans i18nKey="configureImmunizationsPrompt">
```

File: src/widgets/programs/programs-form.component.tsx [hunk 4]
```diff
@@ -229,7 +229,7 @@ export default function ProgramsForm(props: ProgramsFormProps) {
                   onChange={evt => setProgram(evt.target.value)}
                   required
                 >
-                  <option>{t("Choose a program")}:</option>
+                  <option>{t("chooseAProgram", "Choose a program")}:</option>
```
Comment (brandones) [inline src/widgets/programs/programs-form.component.tsx]:
```suggestion
                  <option>{t("chooseProgram", "Choose a program")}:</option>
```

These are like variable names, so you can drop particles

File: src/widgets/programs/programs-form.component.tsx [hunk 5]
```diff
@@ -266,7 +266,7 @@ export default function ProgramsForm(props: ProgramsFormProps) {
                         <svg className="omrs-icon" role="img">
                           <use xlinkHref="#omrs-icon-important-notification"></use>
                         </svg>
-                        <Trans i18nKey="dateErrMsg">
+                        <Trans i18nKey="pleaseEnterADateThatIsEitherOnOrBeforeToday">
```
Comment (brandones) [inline src/widgets/programs/programs-form.component.tsx]:
```suggestion
                        <Trans i18nKey="pastDateErrorMsg">
```
?

File: src/widgets/visit/visit-button-component.tsx [hunk 6]
```diff
@@ -201,7 +201,7 @@ export const EndVisit: React.FC<EndVisitProps> = ({ currentVisit }) => {
   return (
     <div className={styles.visitPromptContainer}>
       <h2>
-        <Trans i18nKey="End Visit Prompt Message">
+        <Trans i18nKey="areYouSureYouWishToEndThisVisit">
```
Comment (brandones) [inline src/widgets/visit/visit-button-component.tsx]:
```suggestion
        <Trans i18nKey="endVisitPrompt">
```

**Review by brandones — APPROVED** [review]
Awesome. Thanks so much!

File: src/ui-components/empty-state/empty-state.component.tsx [hunk 7]
```diff
@@ -19,7 +19,7 @@ export default function EmptyState(props: EmptyStateProps) {
         <p className="omrs-type-body-regular">
           <Trans
             i18nKey="emptyStateText"
-            values={{ displayText: props.displayText }}
+            values={{ displayText: props.displayText.toLowerCase() }}
```
Comment (denniskigen) [inline src/ui-components/empty-state/empty-state.component.tsx:22]: This works fine. The string comes in already translated so what I'm ensuring that it is set to lowercase. Do you still feel a text-transform would be better suited for this?

Comment (brandones) [reply src/ui-components/empty-state/empty-state.component.tsx:22]: This is also fine IMO :+1:

Comment (denniskigen) [reply src/ui-components/empty-state/empty-state.component.tsx:22]: Great, thanks Brandon

**Review by FlorianRappl — APPROVED** [review]
LGTM - just the conflicts need to be resolved. :beers:

**Comment by denniskigen** [comment]
> LGTM - just the conflicts need to be resolved. 🍻

Resolved, thanks @FlorianRappl

**Pull request merged**